packages/web-components/src/progress-bar/progress-bar.styles.ts

```diff
@@ -0,0 +1,154 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorBrandBackground2,
+  colorCompoundBrandBackground,
+  colorNeutralBackground6,
+  colorPaletteDarkOrangeBackground3,
+  colorPaletteGreenBackground3,
+  colorPaletteRedBackground3,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --progress-bar-thickness: 2px;
+    --progress-bar-radius: ${borderRadiusMedium};
+    --progress-bar-color: ${colorCompoundBrandBackground};
+    --progress-speed: 3s;
+    align-items: center;
+    height: var(--progress-bar-thickness);
+    overflow-x: hidden;
+  }
+
+  :host([thickness='large']) {
+    --progress-bar-thickness: 4px;
+  }
+
+  :host([shape='rectangular']) {
+    --progress-bar-radius: 0;
+  }
+
+  :host([validation-state='error']) {
+    --progress-bar-color: ${colorPaletteRedBackground3};
+  }
+
+  :host([validation-state='warning']) {
+    --progress-bar-color: ${colorPaletteDarkOrangeBackground3};
+  }
+
+  :host([validation-state='success']) {
+    --progress-bar-color: ${colorPaletteGreenBackground3};
+  }
+
+  .progress {
+    background-color: ${colorNeutralBackground6};
+    border-radius: var(--progress-bar-radius);
+    width: 100%;
+    height: var(--progress-bar-thickness);
+    display: flex;
+    align-items: center;
+    position: relative;
+  }
+
+  .determinate {
+    background-color: var(--progress-bar-color);
+    border-radius: var(--progress-bar-radius);
+    height: var(--progress-bar-thickness);
+    transition: all 0.2s ease-in-out;
+    display: flex;
+  }
+
+  .indeterminate {
+    height: 6px;
+    border-radius: var(--progress-bar-radius);
+    display: flex;
+    width: 100%;
+    position: relative;
+    overflow: hidden;
+  }
+
+  .indeterminate-indicator-1 {
```

Comment:
nit: the indeterminate indicator breaks the rounded corners of the progress bar:
![image](https://user-images.githubusercontent.com/9615899/212910823-04317ff5-d497-4ef9-b7d7-8e7a32466513.png)